grails-app/services/org/broadinstitute/orsp/IssueService.groovy

```diff
@@ -45,14 +46,21 @@ class IssueService {
             IssueExtraProperty.RESPONSIBLE,
             IssueExtraProperty.REVIEW_CATEGORY,
             IssueExtraProperty.SOURCE,
-            IssueExtraProperty.SUBMISSION_TYPE]
+            IssueExtraProperty.SUBMISSION_TYPE,
+            IssueExtraProperty.PROJECT_TITLE,
```

Comment:
Some history here. For the thing we describe as title in the UI, we actually use the `summary` field on Issue. Issue, originally, corresponded to a Jira Issue since we used Jira as a back end for this app for the first couple years of its life. We then moved to its own database when it outgrew Jira. In Jira, there are a small number of primary fields on the Issue object, and then all sorts of custom fields can be defined that are saved in a separate table. The distinction between where we store things is a bit arbitrary, but in general, really important things are on the `Issue` object itself, while less important things were stored as custom fields. Custom fields became `IssueExtraProperty`.

The point here is that I think we should keep `title`, but move it to the issue directly via a liquibase change. Also, the content of that field should start with the value currently stored in `summary`. We could then use both fields more appropriately in the UI. 

However, this change seems like it should be in a separate PR for clarity.

Reply:
@rushtong We agree, maybe you can create a story for this (If you can) so we can tackle this change in the near future.